tool_start.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
+
+docker-compose start
```

Comment:
Some .sh files seem to be missing line breaks at the end. Otherwise, the code seems fine (I haven't tested it).